niworkflows/interfaces/registration.py

```diff
@@ -88,14 +89,40 @@ class ApplyTOPUPRPT(nrc.RegistrationRC, fsl.ApplyTOPUP):
     output_spec = ApplyTOPUPOutputSpecRPT
 
     def _post_run_hook(self, runtime):
-        self._fixed_image_label = "before"
-        self._moving_image_label = "after"
-        self._fixed_image = index_img(self.inputs.in_files[0], 0)
-        self._moving_image = index_img(self.aggregate_outputs().out_corrected, 0)
-        NIWORKFLOWS_LOG.info('Report - setting before (%s) and after (%s) images',
+        self._fixed_image_label = "corrected"
+        self._moving_image_label = "warped"
```

Comment:
I find this selection of words a bit confusing. Afterall, you apply a non-linear warp filed tot correct the image so technically both images are "warped". That's why I chose 'before' and 'after' because it's so unambiguous. I leave the final decision to your discretion though.